src/main/java/org/folio/linked/data/configuration/json/deserialization/event/DataImportEventDeserializer.java

```diff
@@ -26,6 +29,9 @@ public class DataImportEventDeserializer extends JsonDeserializer<DataImportEven
   private static final String PARSED_RECORD_FIELD = "parsedRecord";
   private static final String CONTENT_FIELD = "content";
   private static final String TENANT_FIELD = "tenant";
+  private static final String DI_INVENTORY_INSTANCE_CREATED = "DI_INVENTORY_INSTANCE_CREATED";
```

Comment:
This is a throw away class. This class with be removed once [UXPROD-4497](https://folio-org.atlassian.net/browse/UXPROD-4497) is implemented.